tests/llama_stack/agents/test_agents.py

```diff
@@ -143,11 +142,12 @@ def test_agents_rag_agent(
         turns_with_expectations = get_torchtune_test_expectations()
 
         # Ask the agent about the inserted documents and validate responses
+        enable_streaming = False
```

Comment:
Can we have some comment here. Something like that without streaming our automation is failing and describe the reason.